include/bluetooth/mesh/time_srv.rst

```diff
@@ -25,9 +25,9 @@ To ensure consistent synchronization, the Time Server model should be configured
 
 .. note::
 
-   To ensure a high level of accuracy, Time Status messages are always transmitted one hop at a time.
-   This means that every mesh node that needs access to the common time reference must be within radio range of at least one other node with the Time Server model.
-   Time Server nodes that have no neighboring Time Servers need to get this information in some other way.
+   To ensure a high level of accuracy, Time Status messages are always transmitted one hop at a time if it is sent as an unsolicited message.
+   This means that every mesh node that needs access to the common time reference without access to the Time Authority must be within the radio range of at least one other node with the Time Server model.
+   Time Server nodes that have no neighboring Time Servers need to get this information in some other way (e.g. getting time from the Time Authority).
```

Comment:
Are there other ways to get the time besides using the Time Authority, or is the Time Authority the only way?

I want to confirm it is correct to use "e.g." instead of "i.e" or rephrasing.

Reply:
Yep, it is possible to use other approaches to synchronize time that is out of the scope of Bluetooth mesh model specification. For instance, if a device has access to the internet it can use NTP servers and etc.